mk/Common.mk

```diff
@@ -712,8 +712,25 @@ SVLIB_LIB    = $(SVLIB_PKG)/../svlib_dpi.so
 SVLIB_CXX    = gcc
 
 svlib: $(SVLIB_PKG)
-	$(SVLIB_CXX) $(SVLIB_CFLAGS) $(SVLIB) $(SVLIB_SRC) -I$(DPI_INCLUDE) -o $(SVLIB_LIB)
+	@echo "$(BANNER)"
+	@echo "Building $(SVLIB_PKG)"
+	@echo "$(BGANNER)"
```

Comment:
`$(BGANNER)`: typo?  Should be `$(BANNER)` IMHO

Reply:
Good catch!  Fixed.